lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -600,9 +601,9 @@ def _section_data_download(course, access):
         settings.FEATURES.get('ENABLE_SPECIAL_EXAMS', False) and
         course.enable_proctored_exams
     )
-
+    section_key = 'data_download_2' if data_download_v2_is_enabled(course_key) else 'data_download'
```

Comment:
Just to make sure, do we have a cleanup ticket? I mean once the code lands on prods and everything works as expected, do we have a ticket to remove the legacy code and waffle flag?

Reply:
The cleanup ticket does not exist at the moment. So I  will create a new ticket for cleanup.

Reply:
second thought. 

Can we also leave the old view as is so that other installations can still use that? we would need to sync up with product.